libs/stokesian_dynamics/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+if(STOKESIAN_DYNAMICS_GPU)
+  cuda_add_library(sd_gpu SHARED sd_gpu.cu)
```

Comment:
`cuda_add_library()` is a CUDA-specific command that doesn't exist when compiling with Clang or HIP. Maybe `add_gpu_library()` could work for you.

Reply:
although, bear in mind `add_gpu_library()` only exists in espresso, it can't make it into the external SD library

Reply:
Hm... I've just done the replacement. For use in EPSREsSo, it works, because stuff that is declared in CMake is carried over and available in the `libs` directory, too.